api/db/build.gradle

```diff
@@ -8,7 +8,7 @@ repositories {
 }
 
 def db_host = System.getenv("DB_HOST") ?: "localhost"
-def db_port = System.getenv("DB_PORT") ?: "3306"
+def db_port = System.getenv("DB_PORT") ?: "3307"
```

Comment:
please add a comment explaining why 3307 is needed, and update `db-cdr/build.gradle` if it needs to match

Reply:
probably worth doing a general grep for 3006 to see what else needs an update

Reply:
We launch the local dockerized DB on 3306, so either one is valid depending on the situation.

Reply:
This fixes the dev deploy. When I attempt to deploy staging, I'll get a better idea about what else needs updating.